src/app/wallet/walletView.nim

```diff
@@ -43,6 +47,12 @@ QtObject:
                           image: image))
     self.endInsertRows()
 
+  proc setDefaultAccount*(self: WalletView, account: string) =
+    self.defaultAccount = account
+
+  method getDefaultAccount*(self: WalletView): string {.slot.} =
+    return self.defaultAccount
```

Comment:
I noticed we've been using `proc` and `method` interchangeably. I guess both can be `{.slot.}` so maybe it makes sense to have some guide lines on when to use which?

Reply:
Found these interesting links

https://matthiashager.com/proc-method-nim
> In general, method should be used over proc when, and only when, you don't know at compile time which object type from your inheritance tree the function will be handling.

http://goran.krampe.se/2014/10/31/nim-and-oo-part-iii/
According to this, using methods incur in a small performance penalty, but really depends on what you are doing. For most real life code, that performance is not noticeable.